package.json

```diff
@@ -33,6 +33,7 @@
     "prettier": "^2.3.0",
     "rate-limiter-flexible": "^2.3.6",
     "sinon-called-with-diff": "^3.1.1",
+    "sinon-chai": "^4.0.0",
```

Comment:
this adds some new syntactic sugar for assertions:
`expect(stub.calledWith(args)).to.be.true` => `expect(stub).to.have.been.calledWith(args)`
and
`expect(stub.called).to.be.true` => `expect(stub).to.have.been.called`